udata/commands/serve.py

```diff
@@ -66,8 +65,6 @@ def serve(info, host, port, reload, debugger, eager_loading, with_threads):
     settings = os.environ.get('UDATA_SETTINGS',
                               os.path.join(os.getcwd(), 'udata.cfg'))
     extra_files = [settings]
-    if reload:
-        extra_files.extend(assets.manifests_paths())
```

Comment:
We do need some hot reload on static files though?

Reply:
The hot reload on static file is provided by `inv assets-watch` rather than this command.